src/cache-utils.ts

```diff
@@ -73,3 +78,16 @@ export function isCacheFeatureAvailable(): boolean {
   );
   return false;
 }
+
+export async function isCacheEnabled() {
```

Comment:
We might split it into two functions: 
`isCacheEnabled` and `isCachePathsExists`